frameworks/person-escort-record/questions/prescribed-medication.yml

```diff
@@ -1,20 +1,16 @@
 type: radio
 question: Do they need or have they been prescribed any medication?
-hint: This includes relevant regular medication that they need or already have with them, as well as anything that they may have been prescribed whilst in custody
+hint: This consists of any medications they have taken within the last 72 hours (including painkillers) or any regular medications they take or may need during the move or once transferred (including long-term moves and time spent at additional locations)
```

Comment:
Just a note on the plural form of "medication" here. Wonder if we should use "medication" over "medications" in this instance?

Reply:
Makes sense to me, this text was copied from the jira ticket, I'll swap it to medication :)